cmd/ddev/cmd/commands_test.go

```diff
@@ -2,6 +2,13 @@ package cmd
 
 import (
 	"fmt"
+	"os"
+	osexec "os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
```

Comment:
This change was not really intended but made by the linter of VS Code. And looking at other packages this looks like a best practise to place interal packages on the top and gh imports afterwards.

Reply:
I always let goland put them where they belong. That's probably what you're seeing. 

I doubt you should be using any linter beyond gofmt, which I think also organizes them.

Reply:
No, gofmt does not organize them as I've seen with my tests. I'm wondering about the difference we have here between your editor and VS Code